src/coreclr/vm/mono/mono_coreclr.cpp

```diff
@@ -69,26 +71,29 @@ typedef intptr_t ManagedStringPtr_t;
 // Generated by UnityEmbedHost.Generator - Commit these changes
 struct HostStruct
 {
+    int32_t (*add_system_assembly_search_path)(const char* raw_path);
     int (*array_length)(MonoArray* array);
     MonoArray* (*array_new)(MonoDomain* domain, MonoClass* klass, guint32 n);
     MonoObject* (*assembly_get_object)(MonoDomain* domain, MonoAssembly* assembly);
     MonoAssembly* (*assembly_loaded)(MonoAssemblyName* aname);
     MonoClass* (*class_from_name)(MonoImage* image, const char* name_space, const char* name, gboolean ignoreCase);
     MonoClass* (*class_from_systemtypeinstance)(MonoObject* systemTypeInstance);
     MonoImage* (*class_get_image)(MonoClass* klass);
+    MonoDomain* (*create_assembly_load_context)(const char* raw_name);
```

Comment:
In the real editor loader, this will be done from managed. + I dislike using MonoDomain* as the return type, it feels wrong

Reply:
> In the real editor loader, this will be done from managed. + I dislike using MonoDomain* as the return type, it feels wrong

what is the alternative you would suggest here?

Reply:
I'd suggest not exposing any native API for managing ALCs :)